lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -21,6 +23,23 @@
     'lti',
 ]
 
+# exclude these in Metrics
+non_student_roles = ['instructor', 'staff']
+
+def get_non_student_list(course_key):
+    # find all user_ids with instructor or staff roles in student_courseaccessrole table
```

Comment:
Comments that pertain to functions are actually a very specific thing called docstrings in python.
They usually take the form
```
def foo(param):
    """
    Docstring here
    """
```
I.e. they're Triple-`"` delimited strings